app/scripts/components/exploration/components/dataset-selector-modal/footer.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { themeVal, glsp } from '@devseed-ui/theme-provider';
+
+import { Button } from '@devseed-ui/button';
+import { pluralize } from '$utils/pluralize';
+
+interface ModalFooterComponentProps {
+  selectedIds: string[]; 
+  onConfirm: () => void; 
+  isFirstSelection: boolean;
+}
+const LayerNumberHighlight = styled.div`
+  border-radius: 100%;
+  background-color: ${themeVal('color.primary')};
+  color: ${themeVal('color.surface')};
+  font-weight: bold;
+  width: 25px;
+  height: 25px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+`;
+
+const LayerResult = styled.div`
+  display: flex;
+  gap: ${glsp(0.5)};
+`;
+
+export default function ModalFooterRender (props:ModalFooterComponentProps) {
+  const { selectedIds, onConfirm, isFirstSelection } = props;
+  return (
+    <>
+      <LayerResult aria-live='polite' className='selection-info'>
+        {selectedIds.length
+          ? <><LayerNumberHighlight>{selectedIds.length} </LayerNumberHighlight> <div>{pluralize({ singular: 'layer', plural: 'layers', count: selectedIds.length})} selected</div></>
+          : 'No data layers selected'}
+      </LayerResult>
+      {!isFirstSelection && (
+        <Button variation='base-text' onClick={close}>
+          Cancel
+        </Button>
```

Comment:
❓ : In what case would this show up? I couldn't trigger it to render even when selecting multiple layers 🤔 Also is it okay for both these buttons to appear? if not, we should make this a ternary instead with the `Add to map` button

Reply:
Ah yes. This should have been in the conditional (Because we allow empty layers now). Thank you for catching this 🙇  but I am not clear on your concerns about both buttons appearing simultaneously. What potential problem do you see?